android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -907,16 +915,26 @@ private WritableArray getAudioTrackInfo() {
         TrackGroupArray groups = info.getTrackGroups(index);
         for (int i = 0; i < groups.length; ++i) {
             Format format = groups.get(i).getFormat(0);
+            String language = format.language != null ? format.language : "";
+            String languageCode = this.getIso2(language);
+
             WritableMap audioTrack = Arguments.createMap();
             audioTrack.putInt("index", i);
             audioTrack.putString("title", format.id != null ? format.id : "");
             audioTrack.putString("type", format.sampleMimeType);
-            audioTrack.putString("language", format.language != null ? format.language : "");
+            audioTrack.putString("language", languageCode);
             audioTracks.pushMap(audioTrack);
         }
         return audioTracks;
     }
 
+    private String getIso2(String iso3) {
+        if (this.localeMap.containsKey(iso3)) {
+            return this.localeMap.get(iso3).toString();
```

Comment:
Not totally convinced on calling `toString()` on Locale object. Maybe a better idea would be to call `getLanguage()` which suppose to return 2 letter language code.
An overall note is that using ISO-2 codes is being discouraged in the Android documentation in favour of ISO-3 codes.

Reply:
Thanks Lukasz. Have tested getLanguage() and it works as expected. Do you know why ISO-2 is being discouraged, and are there any specific cases that we should guard against?

Reply:
As per Android documentation:

> ISO 639 is not a stable standard— some languages' codes have changed.

Reply:
Thanks. I do think we're forced to use ISO-2 in this case as other parts of the app are stuck using ISO-2. Using ISO-3 here would mean that we'd need to implement a ISO-2/ISO-3 conversion somewhere in the app, which would also be brittle